ocaml/database/db_connections.ml

```diff
@@ -15,6 +15,8 @@ module D = Debug.Make(struct let name = "xapi" end)
 module R = Debug.Make(struct let name = "redo_log" end)
 open D
 
+open Db_cache_types
```

Comment:
Could you move this `open`  closer to the code that needs it?
I think only this `dbconn.fsync_enabled` would need it, which can be written as `dbconn.Db_cache_types.fsync_enabled` or `let open Db_cache_types in dbconn.fsync_enabled`